app/app/settings.py

```diff
@@ -50,6 +50,16 @@
     'django.contrib.auth.middleware.AuthenticationMiddleware',
     'django.contrib.messages.middleware.MessageMiddleware',
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
+	'app.corsMiddleware.cors_middleware',
+	'corsheaders.middleware.CorsMiddleware'
+]
+CORS_ORIGIN_ALLOW_ALL = True
+ALLOWED_HOSTS = ['*']
+
+CORS_ORIGIN_WHITELIST = [
```

Comment:
This placeholder needs to be modified.

Reply:
I think this works fine 
See anyway we're not going to deal with CORS once we've hosted the website so it's fine